apps/bluetooth/js/modules/transfer_manager.js

```diff
@@ -175,7 +175,8 @@ define(function(require) {
     _onBluetoothEnabledChanged:
     function bttm__onBluetoothEnabledChanged(enabled) {
       Debug('_onBluetoothEnabledChanged(): enabled = ' + enabled);
-      if (enabled) {
+      // make sure the value is defined
```

Comment:
Maybe need a `console.error()` in that early return block since supposedly the settings should **never** "changed" to `undefined`?